README.md

```diff
@@ -27,10 +27,20 @@ cordova plugin add https://github.com/samzilverberg/cordova-mixpanel-plugin.git
 **window.mixpanel.people:**
 
 - identify(distinctId, onSuccess, onFail)
-- registerPushToken(apnsOrGcmToken, onSuccess, onFail)
+- registerPushId(registrationId, onSuccess, onFail)
```

Comment:
Hi Sam - I was sorta naming to go along with the push plugin naming, i.e. here in their push registration callback https://github.com/phonegap/phonegap-plugin-push#pushonregistration-callback they return an object with a parameter named "registrationId", although I'd probably change it to setPushId instead of registerPushId.

In any case, yeah I think the name you're suggesting works too, but maybe just setPushToken, the Notifications part probably isn't necessary imo.